src/topology/sets/opens.lean

```diff
@@ -50,6 +50,9 @@ instance : has_mem α (opens α) :=
 
 @[simp] lemma mem_coe {x : α} {U : opens α} : (x ∈ (U : set α)) = (x ∈ U) := rfl
 
+@[simp] lemma topological_space.opens.mem_mk {x : α} {U : set α} {h : is_open U} :
```

Comment:
Sorry, I should have mentioned changing the name since this section is inside a namespace:
```suggestion
@[simp] lemma mem_mk {x : α} {U : set α} {h : is_open U} :
```

Reply:
Thanks! I also merged master in case it helps to fix the build error